lib/src/http_session.dart

```diff
@@ -4,21 +4,23 @@ import 'dart:typed_data';
 
 import 'package:http/http.dart' as http;
 import 'package:http/io_client.dart';
+import 'package:cookie_store/cookie_store.dart';
 
 class HttpSession implements IOClient {
   /// Shared http session instance
   static final shared = HttpSession();
 
-  /// Get current headers value
-  Map<String, String> get headers => _headers;
```

Comment:
I think we should keep this getter to be compatible with the older version.

Reply:
It's not really possible to have a useful presistent `headers` field anymore since not all cookies are returned for all requests. Exposing all the cookies through a field like that would encourage using it unsafely and accessing cookies that aren't supposed to be returned with a request.

Reply:
That makes sense. It should be deprecated.